content/en/continuous-deployment/plugin-guide/policy-engine/_index.md

```diff
@@ -0,0 +1,338 @@
+metadata:
+  namespace: opa # Change this to the namespace where policies will live
+  name: opa-configmap-modifier
+roleRef:
+  kind: Role
+  name: configmap-modifier
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: Group
+  name: system:serviceaccounts:opa # Change this to the namespace OPA is installed in
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: opa-deployment
+  namespace: opa # Change this to the namespace OPA is installed in
+  labels:
+    app: opa
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: opa
+  template:
+    metadata:
+      labels:
+        app: opa
+    spec:
+      containers:
+      # WARNING: OPA is NOT running with an authorization policy configured. This
+      # means that clients can read and write policies in OPA. If you are
+      # deploying OPA in an insecure environment, be sure to configure
+      # authentication and authorization on the daemon. See the Security page for
+      # details: https://www.openpolicyagent.org/docs/security.html.
+        - name: opa
+          image: openpolicyagent/opa:{{< param opa-server-version >}}
+          args:
+            - "run"
+            - "--server"
+            - "--addr=http://0.0.0.0:8181"
+          readinessProbe:
+            httpGet:
+              path: /health
+              scheme: HTTP
+              port: 8181
+            initialDelaySeconds: 3
+            periodSeconds: 5
+          livenessProbe:
+            httpGet:
+              path: /health
+              scheme: HTTP
+              port: 8181
+            initialDelaySeconds: 3
+            periodSeconds: 5
+        - name: kube-mgmt
+          image: openpolicyagent/kube-mgmt:0.9
+          args:
+          # Change this to the namespace where you want OPA to look for policies
+            - "--policies=opa"
+          # Configure the OPA server to only check ConfigMaps with the relevant label
+            - "--require-policy-label=true"
+---
+# Create a static DNS endpoint for Spinnaker to reach OPA
+apiVersion: v1
+kind: Service
+metadata:
+  name: opa
+  namespace: opa # Change this to the namespace OPA is installed in
+spec:
+  selector:
+    app: opa
+  ports:
+  - protocol: TCP
+    port: 8181
+    targetPort: 8181
+</pre></code>
+</details>
+
+## Enable the Policy Engine plugin
+
+You have three options for enabling the Policy Engine plugin:
+
+1. [Armory Operator or Spinnaker Operator](#armory-operator-or-spinnaker-operator)
+1. [Spinnaker services local files](#spinnaker-services-local-files) (Spinnaker only)
+1. [Halyard](#halyard) (Spinnaker only)
+
+### Armory Operator or Spinnaker Operator              
+
+You can enable the Policy Engine plugin using the Armory Operator or the Spinnaker Operator and the sample manifest, which is in the [spinnaker-kustomize-patches repository](https://github.com/armory/spinnaker-kustomize-patches/blob/master/armory/patch-policy-engine-plugin.yml).
```

Comment:
It maybe goes without saying, but feels like we should include a note here that the sample manifest requires the use of Kustomize, otherwise they'll need to manually patch their SpinSvc config. We don't currently require the use of Kustomize with the Operator, so it's possible there's someone out there that's rolling these config files by hand :/